### DISCUSSION TITLE
What days/times should we meet?

### DISCUSSION THREAD

**Discussion by jeff-zucker** [solid-contrib/practitioners#4] [Q&A]
Please give your availabiliity in a comment below using UTC (try [worldtime-buddy](https://www.worldtimebuddy.com/) for time conversion).  Please keep in mind that we want something that works in as many regions as possible which pretty much means in the 14:00 to 20:00 range and hopefully at neither extreme.

**Comment by jeff-zucker** [comment]
I'll go first.  I am available

Monday, Thursday, and Friday between 15:00 and 20:00 UTC.

**Comment by ewingson** [comment]
I am available daily from 15:00 to 23:00 UTC (GMT) our Winter time that is.
my aim is to learn basics.

**Comment by NoelDeMartin** [comment]
I am mostly available any day except Thursdays, so I'll probably miss the next meeting. My timezone is CET (UTC +1), and anything that is within 7am to 7pm UTC should work for me. Although if I have to choose I prefer Fridays, since they're the days I work on Solid (I have a day job otherwise, but my schedule is flexible so I should be able to attend).

**Comment by mariha** [comment]
My timezone is CET (UTC+1). I am available from 4:15pm to 7pm UTC with some flexibility to extend that to 8+pm UTC. Every other week I have a call on Tue at 6-8pm UTC and once per month I am not available on Mondays and Fridays (I can give the exact dates if it was needed).

**Comment by YetAnotherJonWilson** [comment]
My schedule can change at any time, but currently, these are the times that I can usually make work:

**Any day at 14:00 UTC and any time on Fridays**,
and,
Mondays: from 17:00 UTC to 20:00 UTC
Tuesdays: from 18:00 UTC to 20:00 UTC
Wednesdays: 16:00 UTC to 20:00 UTC
Thursdays: 16:00 UTC to 20:00 UTC

**Comment by renyuneyun** [comment]
Although probably doesn't matter as it seems, I am available on most of the following time (subject to change; in UK time, i.e. UTC):

- Monday from 15:00
- Tuesday from 16:00
- Friday 14:30-16:00 & 17:00-18:00

I'm also available on many Thursday afternoons from 15:00, but different in the next two weeks.

**Comment by timbot1789** [comment]
I'm generally available Monday, Wednesday, and Thursday 16:00-17:00 UTC or 20:00-21:00 UTC

**Comment by danielbakas** [comment]
Hi everyone! I'm available most days from 16:00-20:00 UTC 😄

I've transcribed our schedules to this [🧑🏽‍💻 Solid Practitioners](https://www.when2meet.com/?22606584-KIcro) When2Meet with our usernames. You can even enter your username (without password) and update them if you want/need. 🤓

Seems like, so far, the best times are:

* Mondays or Fridays @ 17:00 UTC
* Mondays @ 18:00 UTC

Hope this is useful. Cheers!

<img width="331" alt="Captura de pantalla 2023-11-28 a la(s) 10 16 52 p m" src="https://github.com/solid-contrib/practitioners/assets/11587688/47d7ac5a-b6e0-4895-bff3-bb2cafdddb23">

**Comment by pinfold** [comment]
I am available most weeks at the following times:

Mondays: from 16:00 UTC to 1700 UTC and 1800 to 20:00 UTC
Tuesdays: from 17:00 UTC to 20:00 UTC
Wednesday: 18:00 UTC to 19:00 UTC
Friday: 17:00 UTC to 20:00 UTC

**Comment by michielbdejong** [comment]
See @jeff-zucker's [ML post](https://lists.w3.org/Archives/Public/public-solid/2024Jan/0001.html):

> Meetings are every other Monday at 18:00 UTC at
https://meet.jit.si/solid-practitioners. More details and meeting
agendas/notes available in our github repo at
http://github.com/solid-contrib/practitioners.

**Comment by michielbdejong** [comment]
So there will be meetings on 22 Jan, 5 Feb, 19 Feb, ...
It's a bit unclear whether there will be a meeting on 12 Feb (see https://matrix.to/#/!ywIfjdAsxMggBTEMtI:matrix.org/$-3m8NVtOiyRSA43HRxPokBlFfly3TfmXoyvkzYGP7xQ?via=matrix.org&via=gitter.im&via=attendees.fosdem.org)
